webpack.config.development.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable */
+
```

Comment:
We could improve this whole file by switching to webpack-merge or the like, but didn't want to get too crazy off of the bat